pkg/pillar/types/global.go

```diff
@@ -196,6 +196,8 @@ const (
 	LocationCloudInterval GlobalSettingKey = "timer.location.cloud.interval"
 	// LocationAppInterval global setting key
 	LocationAppInterval GlobalSettingKey = "timer.location.app.interval"
+	// NTPSourcesInterval global setting key
+	NTPSourcesInterval GlobalSettingKey = "timer.ntpsources.interval"
```

Comment:
Please mention this here: https://github.com/lf-edge/eve/blob/master/docs/CONFIG-PROPERTIES.md